README.md

```diff
@@ -183,9 +183,10 @@ You must configure each EVM account's private keys and EVM node provider API key
    VALIDATOR_ALCHEMY_ARBITRUM_MAINNET_API_KEY=<your arbitrum mainnet alchemy key>
    VALIDATOR_ALCHEMY_POLYGON_MAINNET_API_KEY=<your polygon mainnet alchemy key>
    VALIDATOR_QUICKNODE_ARBITRUM_NOVA_MAINNET_API_KEY=<your arbitrum nova mainnet quicknode key>
+   VALIDATOR_GLIF_FILECOIN_MAINNET_API_KEY=
    ```
 
-   > Note: there is also an optional `METRICS_HUB_API_KEY` variable; this can be left empty. It's a service (`cmd/metricshub`) that aggregates metrics like `git summary` and pushes them to centralized infrastructure ([GCP Cloud Run](https://cloud.google.com/run)) managed by the core team. If you'd like to have your validator push metrics to this hub, please reach out to the Tableland team, and we may make it available to you. However, this process will further be decentralized in the future and remove this dependency entirely. Additionally, if you'd like to run a validator for the Filecoin Calibration testnet, use the `VALIDATOR_GLIF_FILECOIN_CALIBRATION_API_KEY` key with an empty value, placed in the `docker/deployed/testnet` config since it allows for unauthenticated requests.
+   > Note: there is also an optional `METRICS_HUB_API_KEY` variable; this can be left empty. It's a service (`cmd/metricshub`) that aggregates metrics like `git summary` and pushes them to centralized infrastructure ([GCP Cloud Run](https://cloud.google.com/run)) managed by the core team. If you'd like to have your validator push metrics to this hub, please reach out to the Tableland team, and we may make it available to you. However, this process will further be decentralized in the future and remove this dependency entirely.
```

Comment:
This part about having an empty key for `calibration` testnet still holds, so there is no need to delete it? For the mainnet we have a different key at `VALIDATOR_GLIF_FILECOIN_MAINNET_API_KEY=`

Reply:
umm so the main reason i had it there originally was because the `.env` variables above that section didn't make any mention of FIL mainnet since we didn't support it, yet. so, i had wanted to provide clarity on what was needed in the testets setup since it, basically, replicates what's described for mainnets. 

now that we have a mainnet FIL variable there, i think it's probably fine to drop the mention. plus, the `.env` example has the proper key for testnet-only support.

Reply:
okay, sounds good 👍